app/controllers/answers_controller.rb

```diff
@@ -50,6 +59,15 @@ def update
   end
 
   def update_as_suggested_edit
+    if params[:answer][:body_markdown] == @answer.body_markdown
+      same_body = true
+    end
+
+    if same_body
+      flash[:danger] = 'No changes were saved, there were none to save.'
+      return redirect_to question_path(@answer.parent)
+    end
+
```

Comment:
```suggestion
    if params[:answer][:body_markdown] == @answer.body_markdown
      flash[:danger] = "No changes were saved because you didn't edit the post."
      return redirect_to question_path(@answer.parent)
    end
```